Difficulty installing stormpy

Hello,
I've recently been installing storm+stormpy on multiple machines following the docs and the latest one under conda is giving me some trouble.
Storm is version 1.8.1 and works fine, stormpy (either from master or release 1.8.0) installation fails during the build of `CMakeFiles/storage.dir/src/storage/matrix.cpp.o`. It seems to be some linking problem but my ability to identify where the missing function should be from is limited - any help identifying the problem is appreciated.

Output from `storm --version`:
```
Storm 1.8.1

Date: Mon Sep 11 11:50:47 2023
Command line arguments: --version
Current working directory: /home/jules/bgu/masters/thesis/experiments/stormpy

Version 1.8.1 build from revision 3f74f3e59acfba3b61c686af01a864962d44af97 (clean)
Compiled on Linux 5.4.0-150-generic using gcc 7.5.0 with flags ' -O3 -DNDEBUG -fprefetch-loop-arrays -flto -flto-partition=none -march=native -fomit-frame-pointer'
Linked with GNU Linear Programming Kit v4.65.
Linked with Microsoft Z3 Optimizer v4.4 Build 1 Rev 0.
Linked with CArL.
```

Output from `pip install -ve .` is [attached here](https://github.com/moves-rwth/stormpy/files/12574339/install_log.txt) as it is quite large, important bits seem to be
```
Obtaining file:///home/jules/bgu/masters/thesis/experiments/stormpy
  Running command python setup.py egg_info
  /home/jules/miniconda3/envs/storm/lib/python3.9/site-packages/setuptools/__init__.py:84: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
...
    dist.fetch_build_eggs(dist.setup_requires)
  running egg_info
  creating /tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info
  writing /tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info/PKG-INFO
  writing dependency_links to /tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info/dependency_links.txt
  writing requirements to /tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info/requires.txt
  writing top-level names to /tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info/top_level.txt
  writing manifest file '/tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info/SOURCES.txt'
  reading manifest file '/tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'LICENSE'
  writing manifest file '/tmp/pip-pip-egg-info-9np5u9s7/stormpy.egg-info/SOURCES.txt'
  Preparing metadata (setup.py) ... done
Requirement already satisfied: pycarl>=2.2.0 in /home/jules/bgu/masters/thesis/experiments/pycarl/lib (from stormpy==1.8.0) (2.2.0)
Installing collected packages: stormpy
  Running setup.py develop for stormpy
    Running command python setup.py develop
...
    [  5%] Building CXX object CMakeFiles/storage.dir/src/storage/matrix.cpp.o
    /home/jules/bgu/masters/thesis/experiments/stormpy/src/storage/matrix.cpp: In instantiation of ‘define_sparse_matrix(pybind11::module&, const string&)::<lambda(SparseMatrix<ValueType>&, entry_index<ValueType>)> [with ValueType = double; SparseMatrix<ValueType> = storm::storage::SparseMatrix<double>; entry_index<ValueType> = long unsigned int]’:
    /home/jules/bgu/masters/thesis/experiments/stormpy/src/storage/matrix.cpp:84:37:   required from ‘struct define_sparse_matrix(pybind11::module&, const string&) [with ValueType = double; pybind11::module = pybind11::module_; std::__cxx11::string = std::__cxx11::basic_string<char>]::<lambda(SparseMatrix<double>&, entry_index<double>)>’
    /home/jules/bgu/masters/thesis/experiments/stormpy/src/storage/matrix.cpp:88:10:   required from ‘void define_sparse_matrix(pybind11::module&, const string&) [with ValueType = double; pybind11::module = pybind11::module_; std::__cxx11::string = std::__cxx11::basic_string<char>]’
    /home/jules/bgu/masters/thesis/experiments/stormpy/src/storage/matrix.cpp:132:86:   required from here
    /home/jules/bgu/masters/thesis/experiments/stormpy/src/storage/matrix.cpp:86:29: error: class template argument deduction failed:
                     return std::vector(range.begin(), range.end());
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...
```

--- a/src/storage/matrix.cpp
+++ b/src/storage/matrix.cpp
@@ -83,7 +83,7 @@ void define_sparse_matrix(py::module& m, std::string const& vtSuffix) {
             }, py::return_value_policy::reference, py::keep_alive<1, 0>(), py::arg("row"), "Get row")
         .def("get_rows_for_group", [](SparseMatrix<ValueType>& matrix, entry_index<ValueType> group) {
                 auto range = matrix.getRowGroupIndices(group);
-                return std::vector(range.begin(), range.end());
+                return std::vector<uint64_t>(range.begin(), range.end());
             }, py::arg("row_group"), "Get rows within a row group")
         .def("print_row", [](SparseMatrix<ValueType> const& matrix, entry_index<ValueType> row) {
                 std::stringstream stream;

